app/src/main/java/fr/free/nrw/commons/achievements/Achievements.java

```diff
@@ -1,5 +1,117 @@
 package fr.free.nrw.commons.achievements;
 
+/**
+ * represnts Achievements
+ */
 public class Achievements {
```

Comment:
You could study about `Builder` pattern if you want. It's quite useful with model classes have big constructors.

Reply:
Moreover, is a constructor not needed for this class?